README.md

```diff
@@ -1,16 +1,119 @@
 # getappmap/analyze-action
 
-GitHub action to analyze the changes between two code revisions.
+> To get started with AppMap in GitHub actions, you need to start by installing the [AppMap App on the official GitHub Marketplace](https://github.com/marketplace/get-appmap)
+
+> To see a step-by-step example of how to install this action into your software project, [review the official AppMap Documentation](http://appmap.io/docs/analysis/in-github-actions).
 
-To see a step-by-step example of how to install this action into your software project, [review the official AppMap Documentation](http://appmap.io/docs/analysis/in-github-actions).
+GitHub action to analyze the changes between two code revisions.
 
 AppMap analysis is performed by comparing two AppMap archives - "base" and "head". The "base" archive is created by the
 GitHub Action 
 [`getappmap/archive-action`](https://github.com/getappmap/archive-action/blob/main/action.yml) when code is merged to main.
-The "head" archive is created by this (analyze) Action. The report generated by this Action includes information such as root cause analysis of failed tests,
-a list of API changes, new and resolved analysis findings (anti-patterns and flaws), 
-and lists of new and changed AppMaps. This action will also add code annotations to identify
-the root cause of failed tests and analysis findings.
+The "head" archive is created by this (analyze) Action. The report generated by this Action includes information such as root cause analysis of failed tests, a list of API changes, new and resolved analysis findings (anti-patterns and flaws), 
+and lists of new and changed AppMaps. This action will also add code annotations to identify the root cause of failed tests and analysis findings.
+
+**NOTE** You should not be calling this GitHub Action directly. This action is the main dependent action for the [AppMap Reusable Workflows.](https://appmap.io/docs/reference/github-action.html#reusable-workflows)
```

Comment:
This link will work after this PR is merged.  https://github.com/getappmap/applandinc.github.io/pull/1191

Reply:
Ok. Because this is the action reference, I think we can remove the warning about using it. It’s ok if people don’t want to use our reusable workflow - if they have some reason.